core/src/main/res/values/strings.xml

```diff
@@ -216,6 +216,7 @@
     <string name="download_error_malformed_url">Malformed URL</string>
     <string name="download_error_io_error">IO Error</string>
     <string name="download_error_request_error">Request Error</string>
+    <string name="null_value_error_error">Null Value Error</string>
```

Comment:
`Null` is something developers understand but users will not understand that. It's better to write something like `No podcast was provided that could be shown`.